### ISSUE TITLE
PLEASE NOTE: Sunsetting `openeo-processes-python`

### ISSUE THREAD

**Issue by LukeWeidenwalker** [Open-EO/openeo-processes-python#198]
# PLEASE NOTE: Sunsetting `openeo-processes-python`

**TLDR: EODC is moving away from using the current `openeo-processes-python` repo in production, staff will thus discontinue maintenance of this repository.** 

From the backend refactoring we're currently undergoing we've ended up with a long list of reasons why the current structure doesn't serve our purposes anymore. The main one is that we need our process implementations to be dask-friendly by default. The choice was either releasing a v2, where almost everything looks different or just creating a whole new repo for which we can ensure dask-friendliness from the start. We opted for the latter - this will be called `openeo-processes-dask` and open-source as soon as we have a usable version ready. 

What will happen next:
1) We won't be adding new features or fixing bugs in this repo going forward.
2) Once we've deployed our recent backend refactoring to production, we will revert some of the recent changes that might have caused additional incompatibilities for existing users (e.g. EODC specific parts, packaging changes).
3) We will then wrap this repo up with a final release and shift our development efforts to the new repo.

If anyone is interested in taking over maintenance for the current setup, please let us know below. Otherwise, this repo will enter indefinite hibernation. 

We'd welcome current users to switch to `openeo-processes-dask` with us once it is available - when that time has come, I'll make another announcement in the comments on this issue!

**Comment by LukeWeidenwalker** [comment]
The new repo [openeo-processes-dask](https://github.com/Open-EO/openeo-processes-dask) is now available, so please feel free to start contributing there!
